Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -168,6 +168,20 @@ await container.UpsertItemAsync<TItem>(value, new PartitionKey(value.PartitionKe
             return optimizeBandwidth ? value : response.Resource;
         }
 
+        /// <inheritdoc/>
+        public async ValueTask<IEnumerable<TItem>> UpdateAsync(
+            IEnumerable<TItem> values,
+            CancellationToken cancellationToken = default)
+        {
+            IEnumerable<Task<TItem>> updateTasks =
+                values.Select(value => UpdateAsync(value, cancellationToken).AsTask())
+                    .ToList();
+
+            _ = await Task.WhenAll(updateTasks).ConfigureAwait(false);

```

Comment:
I think we can drop the `_ = ` assignment.

```suggestion
            await Task.WhenAll(updateTasks).ConfigureAwait(false);

```